docs/Notebook-Setup.md

```diff
@@ -16,12 +16,12 @@ other plugins that I don't use but will mention at the bottom.
 
 ## The promise:
 
-\> your friend sends you a jupyter notebook file  
-\> `nvim friends_file.ipynb`  
-\> you see a markdown representation of the notebook, including code outputs and images  
```

Comment:
These spaces were there for a reason, checkout the rendered markdown here: https://github.com/dotpie/molten-nvim/blob/docs/conda_venv_setup/docs/Notebook-Setup.md#the-promise

Whitespace at the end of a line matters in markdown (and this is why we hate markdown)

Reply:
Oh this is mistake! Thank you for mention it with kind explanation!